h2o-core/src/main/java/hex/AUC2.java

```diff
@@ -64,7 +64,12 @@
         double mcc = (tp*tn - fp*fn);
         if (mcc == 0) return 0;
         mcc /= Math.sqrt((tp+fp)*(tp+fn)*(tn+fp)*(tn+fn));
-        assert(Math.abs(mcc)<=1.) : tp + " " + fp + " " + fn + " " + tn;
+        // due tp and tn are double; the MCC could be slightly higher than 1. for example 1.000000000000002
+        double eps = 1e-10;
+        assert(Math.abs(mcc) <= 1. + eps) : tp + " " + fp + " " + fn + " " + tn;
+        if(mcc > 1){
```

Comment:
should we handle the negative case as well?

Reply:
Sure.